internal/pkg/v2/infrastructure/interfaces/db.go

```diff
@@ -0,0 +1,16 @@
+//
```

Comment:
As with V1 (https://github.com/edgexfoundry/edgex-go/blob/master/internal/core/data/interfaces/db.go) this is the specific interface for Core Data to use, so it does belong in internal/core/data/...

Reply:
Thanks for pointing out and that makes sense! Updated the path to `internal/core/data/v2/infrastructure/interfaces/db.go`.